devicetypes/smartthings/zigbee-power-meter.src/zigbee-power-meter.groovy

```diff
@@ -0,0 +1,109 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "Zigbee Power Meter", namespace: "smartthings", author: "SmartThings") {
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Sensor"
+
```

Comment:
So, configure() actually won't be called without capability "Configuration". Unless something has changed in Data Management over the last couple of years.

Reply:
From what I can see, this is still true.

Reply:
god dammit

Reply:
@FenMei007S please open a new PR to add this capability.